### PR TITLE
Configure travis for docker

### PR DESCRIPTION
**Pull request by promehul**
###### Explanation About What Code Achieves:

  - I configured the .travis.yml to include the docker services and I pinged every other service from the main app container

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -11,15 +9,23 @@ python:
...
+    - docker-compose build
+    - docker-compose up -d
+    - docker-compose exec app sh -c "apt-get update && apt-get install -y netcat"
+
 script:
-  - redis-cli ping
-  - make test
+    - docker-compose exec app sh -c "nc -vz cache 6379"
+    - docker-compose exec app sh -c "nc -vz db 5432"
+    - docker-compose exec app sh -c "nc -vz proxy 80"
```
Comment (achillesrasquinha) [inline .travis.yml:28]: Very good. Can you create an API route that does a PING?

Something like.

```
@app.route('/ping')
def ping():
     return "{ 'data': 'pong' }"
```

Then maybe, test this route.

`80` checks the proxy. You might also need to check the internal app working.

And then something like

```
$ docker-compose exec proxy sh -c "curl https://localhost/api/ping"
```

**Review by achillesrasquinha — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (promehul) [reply .travis.yml:28]: Yeah right. @achillesrasquinha so should I write the ping function in a new file in `/candis/api/server/api/ping.py` or do it in `/candis/api/server/api/user.py` .

Comment (promehul) [reply .travis.yml:28]: @achillesrasquinha  Done.

**Comment by promehul** [comment]
@achillesrasquinha please review.

**Comment by achillesrasquinha** [comment]
Good work! 👍

**Pull request merged**